src/routes/embedded/auth/auth/auth.view.tsx

```diff
@@ -20,11 +20,11 @@ export function AuthEmbeddedView() {
       config={[
         {
           label: "Passkey",
-          onClick: () => authenticate("passkey")
+          onClick: () => authenticate("PASSKEYS")
```

Comment:
For some reason, the enums exported from the backend repo only work as types when imported in this project... So I had to hardcode the values for now :\